src/AppHooks.js

```diff
@@ -1,15 +1,36 @@
-import React, { useState } from 'react';
+import React, { useState, useEffect } from 'react';
+import { Routes, Route } from 'react-router-dom';
+import CalculatorHooks from './components/CalculatorHooks';
+import Header from './components/Header';
+import Quote from './components/Quote';
+import Home from './components/Home';
 import './scss/App.scss';
-import Calculator from './components/CalculatorHooks';
 
-const App = () => {
-  const [state] = useState({ theme: 'light' });
-  const { theme } = state;
+const AppHooks = () => {
+  const [theme, setTheme] = useState('Light');
+  useEffect(() => {
+    document.documentElement.classList.add(theme);
+  }, [theme]);
+
+  const handleTheme = () => {
+    if (theme === 'Light') {
+      setTheme('Dark');
+    } else {
+      setTheme('Light');
+    }
+  };
```

Comment:
-  [ ] [OPTIONAL] you can delete the previous `App` component that you still have in your repo since you are using the component called `AppHooks` instead. or instead replace the `AppHooks` with `App`. `AppHooks` isn't a very reliable name for this component.

Reply:
Thanks for noticing this @MrBrN197,
I kept these class components for future reference.